oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -87,13 +87,19 @@
    *
    * @param transportFactory The Http transport factory
    * @return the credential instance
+   * @deprecated Use {@link #newBuilder()} instead to construct an instance using the builder. This
```

Comment:
Probably this description (also applies to other deprecated consturctors documentation) is unnecessarily verbose. "to construct an instance using the builder" - it is obvious, and does not need to be documented. "deleted or made private" - made private is same as deleted from point of view of client code.

Reply:
I can make it less verbose.

You are correct that it is the same from the client code point of view, but documentation is also for maintainers too.